cuqi/geometry/_geometry.py

```diff
@@ -732,7 +733,28 @@ def par2fun(self, p):
 
     def fun2par(self, funvals):
         """The function to parameter map used to map function values back to parameters, if available."""
```

Comment:
Please add here that the a projection onto the coefficients is used and so the signal may not be fully represented but it will be the closest approximation in that expansion, or something like that.